src/engagement_db_to_analysis/traced_data_filters.py

```diff
@@ -60,15 +60,50 @@ def rqa_time_range_filter(user, messages_traced_data, pipeline_config):
         else:
             filtered.append(td)
 
-    log.info(f"Filtered out messages{time_range_log}. "
-             f"Returning {len(filtered)}/{len(messages_traced_data)} messages.")
+    log.info(f"Returning {len(filtered)}/{len(messages_traced_data)} messages...")
 
     return filtered
 
 
+def filter_test_participants(user, participants_traced_data_map, test_contacts):
+    """
+    Filters a dict of participants who are not in pipeline_config.test_contacts e.g AVF/Aggregator staff
+
+    :param participants_traced_data_map: A dict of TracedData individuals objects to filter.
+    :type participants_traced_data_map: dict of TracedData
+    :param test_contacts: a list containing test participant uids.
+    :type test_contacts: list of str
+    :return: Filtered dict.
+    :rtype: dict of participant_uid -> TracedData
+    """
+    log.debug("Filtering out test participants data...")
+    filtered = {}
+
+    for uid, trace_data in participants_traced_data_map.items():
+        if uid in test_contacts:
+            continue
+
+        trace_data.append_data(trace_data, Metadata(user, Metadata.get_call_location(), time.time()))
```

Comment:
```suggestion
        trace_data.append_data(trace_data, Metadata(user, Metadata.get_call_location(), TimeUtils.utc_now_as_iso_string()))
```
time.time() was used before we understood how to safely create timestamps in a way which was human readable too

Reply:
updated thanks